tests/e2e/conftest.py

```diff
@@ -109,3 +122,102 @@ def _f(cluster: str) -> Iterator[None]:
                 )
 
     return _f
+
+
+@pytest.fixture
+def dockerhub_auth_secret(_neuro_client: neuro_api.Client) -> Iterator[Secret]:
+    secret_name = f"{KANIKO_AUTH_PREFIX}_{uuid.uuid4().hex}"
+    auth_data = _construct_registy_auth(
+        # Why not v2: https://github.com/GoogleContainerTools/kaniko/pull/1209
+        registry_uri="https://index.docker.io/v1/",
+        username=os.environ["DOCKER_CI_USERNAME"],
+        password=os.environ["DOCKER_CI_TOKEN"],
+    )
+    secret = Secret(secret_name, auth_data)
+    try:
+        run_async(_neuro_client.secrets.add(secret_name, auth_data.encode()))
+        yield secret
+    finally:
+        run_async(_neuro_client.secrets.rm(secret_name))
+
+
+@pytest.fixture
+def project_dir() -> Iterator[Path]:
```

Comment:
pytest already has a `tempdir` fixture which can be used in the test as follows:

```python
def test_sample(tmpdir):
    with tmpdir.mkdir('test').as_cwd():
        ...
```

No need for custom fixture here

Reply:
The cool thing about the `project_dir` fixture is that we might use it only in a single place - `cli_runner` fixture, otherwise we will need to import it into each test and perform `cwd` switches.

Or maybe I've got your idea wrong, if so - let's discuss it via voice.

Reply:
As discussed during the call, I've investigated the proposed fixture's code (and a couple of neighboring fixtures), but unfortunately, it isn't possible to apply the fixture without refactoring other tests (not only the cli_runner fixture).
Therefore, I've decided to leave it as-is for now.